Jenkinsfile

```diff
@@ -1,3 +1,7 @@
+properties properties: [
+    disableConcurrentBuilds(),
+]
+
 try {
     node('slave') {
```

Comment:
I think I'd just put a line after this one with this:

    options { disableConcurrentBuilds() }

Seems a little cleaner, but I think what you have should work too.